<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisionApp</class>
 <widget class="QMainWindow" name="VisionApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>630</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vision System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color:white;
	background-color: rgb(50, 50, 50);
}</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(50, 50, 50);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox{
	border:2px dashed black;
	color: rgb(255, 255, 255);
	background-color: rgb(110, 110, 110);
}

QLabel{
	color: rgb(255, 255, 255);
	background-color:none;
}

QLineEdit{
	color:rgb(6, 6, 6);
	background-color:none;
}</string>
         </property>
         <property name="title">
          <string>Current Program :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Current Recipe
Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Current Reipe
Numer :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>PLC
Command :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Application
Status :</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignmentFlag::AlignLeft">
           <widget class="QPushButton" name="pushButton_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	
	background-color: rgb(255, 0, 0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="OutputImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color:white;
}</string>
         </property>
         <property name="text">
          <string>Here The Image will be shown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
	border:2px dashed black;
	color: rgb(255, 255, 255);
	background-color: rgb(110, 110, 110);
}

QLabel{
	color: rgb(255, 255, 255);
	background-color:none;
}

QLineEdit{
	color:rgb(16, 255, 12);
	background-color:none;
}</string>
      </property>
      <property name="title">
       <string>Running Parameters</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="topMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Camera Parameter</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Model Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Model Serial No :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>InterFace Type :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>lineEdit</zorder>
         <zorder>label_2</zorder>
         <zorder>label_3</zorder>
         <zorder>label_4</zorder>
         <zorder>lineEdit_2</zorder>
         <zorder>lineEdit_3</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Cycle Parameter</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color:black;
	background-color: rgb(85, 255, 0);
}</string>
               </property>
               <property name="text">
                <string>---</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
		color:black;
	background-color: rgb(255, 88, 0);
}</string>
               </property>
               <property name="text">
                <string>---</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:black;
	background-color: rgb(255, 255, 0);
}</string>
            </property>
            <property name="text">
             <string>---</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Operations</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(249, 249, 249);
}

QPushButton::hover{
	background-color: rgb(130, 130, 130);
}

QPushButton::checked{
	background-color: rgb(124, 255, 64);
}</string>
            </property>
            <property name="text">
             <string> Start Process        </string>
            </property>
            <property name="icon">
             <iconset resource="StaticAssets.qrc">
              <normaloff>:/Icons/Assets/Icons/icons8-start-64.png</normaloff>:/Icons/Assets/Icons/icons8-start-64.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(249, 249, 249);
}

QPushButton::hover{
	background-color: rgb(130, 130, 130);
}</string>
            </property>
            <property name="text">
             <string>Manual Trigger</string>
            </property>
            <property name="icon">
             <iconset resource="StaticAssets.qrc">
              <normaloff>:/Icons/Assets/Icons/icons8-light-switch-64 (1).png</normaloff>:/Icons/Assets/Icons/icons8-light-switch-64 (1).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(249, 249, 249);
}

QPushButton::hover{
	background-color: rgb(130, 130, 130);
}</string>
            </property>
            <property name="text">
             <string>Open Teach Menu</string>
            </property>
            <property name="icon">
             <iconset resource="StaticAssets.qrc">
              <normaloff>:/Icons/Assets/Icons/icons8-abc-block-100.png</normaloff>:/Icons/Assets/Icons/icons8-abc-block-100.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreateOrModifyRecipe"/>
    <addaction name="actionLoad_Image_2"/>
    <addaction name="actionSave_Current_Image_2"/>
   </widget>
   <widget class="QMenu" name="menuTeach">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Teach</string>
    </property>
    <addaction name="actionOpen_Teaching_Window"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpen_Device_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTeach"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionSave_Current_Image">
   <property name="text">
    <string>Load Program</string>
   </property>
  </action>
  <action name="actionCreateOrModifyRecipe">
   <property name="text">
    <string>Create Or Modify Recipe</string>
   </property>
  </action>
  <action name="actionOpen_Teaching_Window">
   <property name="text">
    <string>Open Teaching Window</string>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionOpen_Device_Settings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Device Settings</string>
   </property>
  </action>
  <action name="actionLoad_Image_2">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionSave_Current_Image_2">
   <property name="text">
    <string>Save Current Image</string>
   </property>
  </action>
  <action name="actionPLC_Device_Settings">
   <property name="text">
    <string>PLC Device Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="StaticAssets.qrc"/>
 </resources>
 <connections/>
</ui>
